apps/src/lib/tools/jsinterpreter/JSInterpreter.js

```diff
@@ -1046,7 +1045,8 @@ export default class JSInterpreter {
    * Returns the current interpreter state object.
    */
   getCurrentState() {
-    return this.interpreter && this.interpreter.peekStackFrame();
+    const currentInterpreter = this.currentEvalInterpreter || this.interpreter;
+    return currentInterpreter && currentInterpreter.peekStackFrame();
```

Comment:
`getCurrentState()` (used by `nativeIsAsync` functions) can now be used properly while using the `evalInterpreter` - allows calls like `me.overlap(g)` to work in the console.